corehq/messaging/scheduling/migrations/0030_remove_connectmessagesurveycontent_message_and_more.py

```diff
@@ -23,13 +23,13 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='connectmessagesurveycontent',
             name='expire_after',
-            field=models.IntegerField(default=django.utils.timezone.now),
+            field=models.IntegerField(default=1),
             preserve_default=False,
         ),
         migrations.AddField(
             model_name='connectmessagesurveycontent',
             name='form_unique_id',
-            field=models.CharField(default=1, max_length=126),
+            field=models.CharField(default="1", max_length=126),
```

Comment:
Why not just allow this to be blank? Or does 1 have a meaning?

Reply:
Nope, this will be unnecessary once we combine the migrations as you mentioned above, we just needed _any_ value to have a valid migration, even though that table was empty when I made this one, so it is never used.